pages/common/babel.md

```diff
@@ -0,0 +1,35 @@
+# babel
+
+> A transpiler for writing next generation JavaScript.
```

Comment:
I think the language can be improved as - `A transpiler which converts code from JavaScript ES6/ES7 syntax to ES5 syntax.`. Although it supports other targets too. But as the most general case, this is what it is used for.